docs/sql/statements/insert.rst

```diff
@@ -119,14 +126,22 @@ Parameters
 ==========
 
 :table_ident:
-  The identifier (optionally schema-qualified) of an existing table.
+    The identifier (optionally schema-qualified) of an existing table.
 
 :column_ident:
-  The name of a column or field in the table pointed to by *table_ident*.
+    The name of a column or field in the table pointed to by *table_ident*.
 
 :expression:
-  An expression or value to assign to the corresponding column.
+    An expression or value to assign to the corresponding column.
 
 :query:
-  A query (SELECT statement) that supplies the rows to be inserted.
-  Refer to the ``SELECT`` statement for a description of the syntax.
+    A query (``SELECT`` statement) that supplies the rows to be inserted.
+    Refer to the ``SELECT`` statement for a description of the syntax.
+
+:output_expression:
+    An expression to be computed and returned by the ``INSERT`` command
+    after each row is updated. The expression can use any column names
+    of the table or use ``*`` to return all columns.
```

Comment:
Are system columns not supported? Otherwise maybe add the same as here? https://github.com/crate/crate/commit/289d32dbf9a6bb8b7cfd0b1645ffea368008787f

Reply:
good catch, will do 👍